proposals/0376-function-back-deployment.md

```diff
@@ -2,7 +2,7 @@
 
 * Proposal: [SE-0376](0376-function-back-deployment.md)
 * Author: [Allan Shortlidge](https://github.com/tshortli)
-* Implementation: [apple/swift#41271](https://github.com/apple/swift/pull/41271), [apple/swift#41348](https://github.com/apple/swift/pull/41348), [apple/swift#41416](https://github.com/apple/swift/pull/41416), [apple/swift#41612](https://github.com/apple/swift/pull/41612) as the underscored attribute `@_backDeploy` 
+* Implementation: Available as `@_backDeploy(before:)` in Swift 5.8
```

Comment:
I think it's actually useful to keep the implementation PRs here!

Reply:
```suggestion
* Implementation: [apple/swift#41271](https://github.com/apple/swift/pull/41271), [apple/swift#41348](https://github.com/apple/swift/pull/41348), [apple/swift#41416](https://github.com/apple/swift/pull/41416), [apple/swift#41612](https://github.com/apple/swift/pull/41612) as the underscored attribute `@_backDeploy`
```

Reply:
Ok, they felt incomplete to me because there have been a lot of changes since then but I don't mind keeping them.